services/userService.js

```diff
@@ -0,0 +1,26 @@
+const bcrypt = require("bcrypt");
+
+const userDao = require("../models/userDao");
+
+const signUp = async (email, password, name, phoneNumber) => {
+  const saltRounds = 12;
+
+  const makeHash = async (password, saltRounds) => {
+    return await bcrypt.hash(password, saltRounds);
+  };
+
+  const hashedPassword = await makeHash(password, saltRounds);
+
+  const createUser = await userDao.createUser(
+    email,
+    hashedPassword,
+    name,
+    phoneNumber
+  );
+
+  return createUser;
+};
```

Comment:
```suggestion
  return await userDao.createUser(
    email,
    hashedPassword,
    name,
    phoneNumber
  );
};
```
이 또한 변수 선언 필요없이 바로 return 해주면 되겠습니다.